corehq/apps/users/views/mobile/users.py

```diff
@@ -1084,8 +1084,10 @@ def form(self):
                 require_email=self.invitation.require_email)
 
     def get_context_data(self, **kwargs):
+        domain_obj = Domain.get_by_name(self.domain)
```

Comment:
the domain should already be accessible via `self.domain_object` since this class extends [DomainViewMixin](https://github.com/dimagi/commcare-hq/blob/3d2b1b94606452a52ad2533e32bd8bcaa9acfaac/corehq/apps/domain/views.py#L187)